models/customers.js

```diff
@@ -0,0 +1,36 @@
+var app = require("../app");
+var db = app.get("db");
+
+var Customers = function(id) {
+  this.id = id;
+};
+
+Customers.all = function(callback) {
+  db.customers.find(function(error, customers) {
+    if(error || !customers) {
+      callback(error || new Error("Could not retrieve customers"), undefined);
```

Comment:
There may be a situation where !customers happens because there are no customers in the database yet. A bit nitpicky but it may be worth returning a different error in this situation (ex: "There are no customers")